src/host/EspNowHost.h

```diff
@@ -73,6 +75,9 @@ class EspNowHost {
   typedef std::function<std::optional<FirmwareUpdate>(uint64_t mac_address, uint32_t firmware_version)>
       FirmwareUpdateAvailable;
 
+  typedef std::function<std::optional<EspNowConfigEnvelope>(uint64_t mac_address, uint8_t config_version)>
+      ConfigUpdateAvailable;
```

Comment:
```suggestion
      ConfigurationAvailable;
```
I think we can drop the `Update` here and indicate that "a configuration is available" (same goes for all variables etc as well ofc)

Reply:
fixed